### PR TITLE
implement license manager v2

### PR DESCRIPTION
**Pull request by moorscode**
Unified Yoast SEO name in the plugin.
Applied uniformed plugin name.

Uses the latest version of the License Manager available.
Adds checks to see if the License Manager is available before using it.

Plugin should be able to be activated without Yoast SEO or Yoast SEO Premium to be present.
Whenever Yoast SEO or Yoast SEO Premium is active the License Manager should be used.

### CODE REVIEW

**Comment by rensw90** [comment]
@moorscode, the build failed

File: wpseo-woocommerce.php [hunk 1]
```diff
@@ -131,7 +134,10 @@ public function __construct() {
 		}
 		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
 
-		add_action( 'admin_init', array( $this, 'init_beacon' ) );
+		// Only initialise beacon when the License Manager is present.
```
Comment (andizer) [inline wpseo-woocommerce.php]: We use US-English thus initialise should become initialize.

File: wpseo-woocommerce.php [hunk 2]
```diff
@@ -337,20 +344,20 @@ public function admin_panel() {
 		if ( $wpseo_options['breadcrumbs-enable'] === true ) {
 			echo '
 		<h2>' . __( 'Breadcrumbs', 'yoast-woo-seo' ) . '</h2>
-		<p>' . sprintf( __( 'Both WooCommerce and WordPress SEO by Yoast have breadcrumbs functionality. The WP SEO breadcrumbs have a slightly higher chance of being picked up by search engines and you can configure them a bit more, on the %1$sInternal Links settings page%2$s. To enable them, check the box below and the WooCommerce breadcrumbs will be replaced.', 'yoast-woo-seo' ), '<a href="' . esc_url( admin_url( 'admin.php?page=wpseo_internal-links' ) ) . '">', '</a>' ) . "</p>\n";
+		<p>' . sprintf( __( 'Both WooCommerce and Yoast SEO have breadcrumbs functionality. The Yoast SEO breadcrumbs have a slightly higher chance of being picked up by search engines and you can configure them a bit more, on the %1$sInternal Links settings page%2$s. To enable them, check the box below and the WooCommerce breadcrumbs will be replaced.', 'yoast-woo-seo' ), '<a href="' . esc_url( admin_url( 'admin.php?page=wpseo_internal-links' ) ) . '">', '</a>' ) . "</p>\n";
```
Comment (andizer) [inline wpseo-woocommerce.php]: Please put `Yoast` outside the language string.

File: wpseo-woocommerce.php [hunk 3]
```diff
@@ -337,20 +344,20 @@ public function admin_panel() {
...
 			$this->checkbox( 'breadcrumbs', __( 'Replace WooCommerce Breadcrumbs', 'yoast-woo-seo' ) );
 		}
 
 		echo '
 		<br class="clear"/>
 		<h2>' . __( 'Admin', 'yoast-woo-seo' ) . '</h2>
-		<p>' . __( 'Both WooCommerce and WordPress SEO by Yoast add columns to the product page, to remove all but the SEO score column from Yoast on that page, check this box.', 'yoast-woo-seo' ) . "</p>\n";
-		$this->checkbox( 'hide_columns', __( 'Remove WordPress SEO columns', 'yoast-woo-seo' ) );
+		<p>' . __( 'Both WooCommerce and Yoast SEO add columns to the product page, to remove all but the SEO score column from Yoast SEO on that page, check this box.', 'yoast-woo-seo' ) . "</p>\n";
```
Comment (andizer) [inline wpseo-woocommerce.php]: `Yoast SEO` should be outside the string.

File: wpseo-woocommerce.php [hunk 4]
```diff
@@ -337,20 +344,20 @@ public function admin_panel() {
...
+		$this->checkbox( 'hide_columns', __( 'Remove Yoast SEO columns', 'yoast-woo-seo' ) );
```
Comment (andizer) [inline wpseo-woocommerce.php]: `Yoast SEO` should be outside the string.

File: wpseo-woocommerce.php [hunk 5]
```diff
@@ -337,20 +344,20 @@ public function admin_panel() {
...
 
 		echo '
 		<br class="clear"/>
-		<p>' . __( 'Both WooCommerce and WordPress SEO by Yoast add metaboxes to the edit product page, if you want WooCommerce to be above WordPress SEO, check the box.', 'yoast-woo-seo' ) . "</p>\n";
-		$this->checkbox( 'metabox_woo_top', __( 'Move WooCommerce Up', 'yoast-woo-seo' ) );
+		<p>' . __( 'Both WooCommerce and Yoast SEO add metaboxes to the edit product page, if you want WooCommerce to be above Yoast SEO, check the box.', 'yoast-woo-seo' ) . "</p>\n";
```
Comment (andizer) [inline wpseo-woocommerce.php]: `Yoast SEO` should be outside the string.

File: wpseo-woocommerce.php [hunk 6]
```diff
@@ -769,7 +779,7 @@ public function twitter_enhancement() {
  * @since 1.0.1
  */
 function yoast_wpseo_woocommerce_missing_error() {
-	echo '<div class="error"><p>' . sprintf( __( 'Please %sinstall &amp; activate WordPress SEO by Yoast%s and then enable its XML sitemap functionality to allow the WooCommerce SEO module to work.', 'yoast-woo-seo' ), '<a href="' . esc_url( admin_url( 'plugin-install.php?tab=search&type=term&s=wordpress+seo&plugin-search-input=Search+Plugins' ) ) . '">', '</a>' ) . '</p></div>';
+	echo '<div class="error"><p>' . sprintf( __( 'Please %sinstall &amp; activate Yoast SEO%s and then enable its XML sitemap functionality to allow the WooCommerce SEO module to work.', 'yoast-woo-seo' ), '<a href="' . esc_url( admin_url( 'plugin-install.php?tab=search&type=term&s=wordpress+seo&plugin-search-input=Search+Plugins' ) ) . '">', '</a>' ) . '</p></div>';
```
Comment (andizer) [inline wpseo-woocommerce.php]: `Yoast SEO` should be outside the string.

File: wpseo-woocommerce.php [hunk 7]
```diff
@@ -787,7 +797,7 @@ function yoast_wpseo_woocommerce_wordpress_upgrade_error() {
  * @since 1.0.1
  */
 function yoast_wpseo_woocommerce_upgrade_error() {
-	echo '<div class="error"><p>' . __( 'Please upgrade the WordPress SEO plugin to the latest version to allow the WooCommerce SEO module to work.', 'yoast-woo-seo' ) . '</p></div>';
+	echo '<div class="error"><p>' . __( 'Please upgrade the Yoast SEO plugin to the latest version to allow the WooCommerce SEO module to work.', 'yoast-woo-seo' ) . '</p></div>';
```
Comment (andizer) [inline wpseo-woocommerce.php]: `Yoast SEO` should be outside the string.

**Comment by andizer** [comment]
CR done 👷

**Comment by terw-dan** [comment]
Acceptance :+1:

**Pull request merged**